pkg/runs/runsGet_test.go

```diff
@@ -1365,28 +1391,28 @@ func TestRunsGetActiveRunsBuildsQueryCorrectly(t *testing.T) {
 	mockTimeService := utils.NewMockTimeService()
 
 	// When...
-	err := GetRuns(runName, age, requestor, result, shouldGetActive, outputFormat, mockTimeService, mockConsole, apiServerUrl, apiClient)
+	err := GetRuns(runName, age, requestor, result, shouldGetActive, outputFormat, mockTimeService, mockConsole, apiServerUrl, apiClient, group)
 
 	// Then ...
 	assert.Nil(t, err)
 }
 
-
 func TestRunsGetWithNextCursorGetsNextPageOfRuns(t *testing.T) {
 
 	// Given ...
 	pages := make(map[string][]string, 0)
-	pages[""] = []string{ RUN_U456 }
-	pages["page2"] = []string{ RUN_U456 }
+	pages[""] = []string{RUN_U456}
+	pages["page2"] = []string{RUN_U456}
 	pages["page3"] = []string{}
-	nextPageCursors := []string{ "page2", "page3" }
+	nextPageCursors := []string{"page2", "page3"}
 
 	age := ""
 	runName := "U456"
 	requestor := ""
 	result := ""
 	shouldGetActive := false
 	pageSize := 1
+	group := ""
```

Comment:
I can't see any unit tests here that pass a value in for group, so the group logic isn't being tested